test/test_interpolations.jl

```diff
@@ -46,7 +46,18 @@
 
         # Note that not every element formulation exists for every order and dimension.
         applicable(Ferrite.getlowerorder, interpolation) && @test typeof(Ferrite.getlowerorder(interpolation)) <: Interpolation{ref_shape,func_order-1}
-
+    @testset "transform face points" begin
+        ref_coord = Ferrite.reference_coordinates(Lagrange{ref_shape, 1}())
+        for face in 1:nfaces(interpolation)
+            face_nodes = Ferrite.faces(ref_shape)[face]
+            for node in face_nodes
+                coord = ref_coord[node]
```

Comment:
Perhaps a "harder" test would be a linear combination of coordinates on the face, to avoid just checking exactly the reference coordinates?